tests/std/tests/P0220R1_string_view/test.cpp

```diff
@@ -1088,6 +1088,13 @@ static_assert(u8"abc"sv[1] == u8'b');
 static_assert(noexcept(u8"abc"sv));
 #endif // __cpp_char8_t
 
+// P2166R1 Prohibiting basic_string And basic_string_view Construction From nullptr
+#if _HAS_CXX23
+static_assert(!is_constructible_v<string_view, nullptr_t>, "constructing string_view from nullptr_t is prohibited");
+static_assert(!is_constructible_v<string, nullptr_t>, "constructing string from nullptr_t is prohibited");
+static_assert(!is_assignable_v<string&, nullptr_t>, "assigning string to nullptr_t is prohibited");
+#endif

```

Comment:
Should be commented - our convention isn't totally rigorous, there's a tracking issue about this, but we generally comment all `#endif`s except when they control literally one line and there are no nearby directives. (This makes it easier to expand code in the future, without the risk of forgetting to add a comment.)